rackcity/api/serializers/user_serializers.py

```diff
@@ -14,6 +14,8 @@ def custom_signup(self, request, user):
 
 
 class UserSerializer(serializers.ModelSerializer):
+    is_admin = serializers.BooleanField(source='is_staff')
+
```

Comment:
Renamed this from `is_staff` to `is_admin` in backend & front end

Reply:
yay